indigo/distribution.yaml

```diff
@@ -780,6 +780,18 @@ repositories:
       url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
       version: master
     status: maintained
+  bowpmap_ros-release:
```

Comment:
You might want to use the repo name without the `release`suffix here. If you change this and the repo contains only a single package with the same name you will also need to remove the `packages` entry.